src/components/common/view/TopBar.js

```diff
@@ -59,7 +60,9 @@ const TopBar = ({
           if children=undefined and hideBalance=true, nothing will be rendered
           */}
         <Section.Text style={styles.balance}>
-          {!hideBalance && <BigGoodDollar style={styles.bigGoodDollar} number={balance} />}
+          {!hideBalance && (
+            <BigGoodDollar style={styles.bigGoodDollar} number={balance} unit={isNativeToken ? token : undefined} />
```

Comment:
we should have a unit for native tokens also. celo, eth,fuse

Reply:
what do You mean ? we pass "G$" "FUSE" "CELO" "ETH" "GETH" as `unit` props, it changes formatting depending of token

Reply:
I just mean that the display should say "100 ETH" , "100 G$", "100 CELO"
that's all, if it does that then fine.

Reply:
https://github.com/GoodDollar/GoodDAPP/pull/4019#discussion_r1273333889